lib/sles4sap_publiccloud.pm

```diff
@@ -831,4 +837,105 @@ sub list_cluster_nodes {
     die 'None of the hosts are currently part of existing cluster' unless @cluster_nodes;
 }
 
+=head2 get_hana_database_info
+
+    Parses "hdbsql" command output.
+    Returns 1 if hana database is online, 0 otherwise.
+
+=back
+=cut
+
+sub get_hana_database_info {
+    my ($self, $password_db, $instance_id) = @_;
+    my $hdb_cmd = "hdbsql -u SYSTEM -p $password_db -i $instance_id 'SELECT * FROM SYS.M_DATABASES;'";
+
+    sleep 30;
+    eval {
+        my $output_cmd = $self->run_cmd(cmd => $hdb_cmd, runas => get_required_var("SAP_SIDADM"));
+    };
+    if ($@) {
+        record_info('HANA DB OFFLINE', "Hana database in primary node is offline. Here the error \n$@");
+        return 0;
+    }
+    else {
+        return 1;
+    }
+}
+
+=head2 is_hana_database_offline
+
+    Check if hana DB is offline.
+    Returns 1 if hana database is online, 0 if offline
```

Comment:
This is the same description as in the previous function `get_hana_database_info`. Are both functions doing the same? If they are, why not drop one of them and keep the other? If the functions are not doing the same, then their description cannot be the same.

Reply:
They are not the same one get the status and the other one check the status and wait

Reply:
Perfect. Update the documentation so it's clear where the difference lies.